velox/functions/prestosql/aggregates/tests/AggregationFunctionRegTest.cpp

```diff
@@ -54,6 +54,24 @@ TEST_F(AggregationFunctionRegTest, prefix) {
   });
 }
 
+TEST_F(AggregationFunctionRegTest, orderSensitive) {
+  // Remove all functions and check for no entries.
+  clearAndCheckRegistry();
+
+  std::set<std::string> orderInSensitiveFunction = {
```

Comment:
orderInSensitiveFunction -> nonOrderSensitiveFunctions (make sure to add 's' at the end since this is a set of multiple functions)

Reply:
Done.